server/utils/validation.js

```diff
@@ -0,0 +1,102 @@
+const QuestionTypes = require('./questionTypes');
+
+/** Returns whether the question is a valid short answer question. */
+function isValidShortAnswer(question) {
+  if (!question) {
+    return false;
+  }
+
+  if (!question.title) {
+    return false;
+  }
+
+  if (!question.possibleResponses || !(question.possibleResponses instanceof Array)) {
+    return false;
+  }
+
+  if (question.possibleResponses.length !== 1) {
```

Comment:
could the following ifs all be connect with or statements? I don't know if that's a bad practice but it seems like it would be easier to read.

Reply:
Same with the functions below. Also not a must just a note.

Reply:
good point, I'll see what I can do.

Reply:
I'll try to address this in the third PR #36 -- don't want to create a merge conflict.